### DISCUSSION TITLE
Have --spectrogram to work as --transcode for existing folders

### DISCUSSION THREAD

**Discussion by Asparghus** [RogueOneEcho/caesura#114] [Ideas]
When you run a batch command on a queue that had been previously processed it can identify existing transcode folders (DBUG * Found existing FLAC transcode), so it doesn't run again.

Would be possible to have the same for the --spectrogram command? If the SPECTROGRAM folder is present, it skips the command.

**Comment by RogueOneEcho** [comment]
For transcodes it's trivial as the `.torrent` file is created after transcoding and that is used to validate that the transcode is still intact.

Spectrograms are a little trickier as there's a chance the spectrogram command is cancelled part way through, or the user first runs `full` then wants `zoom` in those cases you would want to spectrogram command to re-run.

A simple solution will be to:
- [ ] generate a hidden `.full.complete` or `.zoom.complete` file inside the spectrogram directory after completion.
- [ ] check this file exists and if so skip

**Reply by RogueOneEcho** [reply]
I opted to ignore the added complexity and have just implemented it as a simple output directory check.

Implemented in: 758f19a2
